tests/e2e/env/setup.sh

```diff
@@ -234,17 +237,17 @@ fi
 echo "Setting up WooCommerce Payments..."
 if [[ "0" == "$(cli wp option list --search=woocommerce_woocommerce_payments_settings --format=count)" ]]; then
 	echo "Creating WooCommerce Payments settings"
-	cli wp option add woocommerce_woocommerce_payments_settings --format=json '{"enabled":"yes"}'
+	cli wp option set woocommerce_woocommerce_payments_settings --format=json '{"enabled":"yes"}'
 else
 	echo "Updating WooCommerce Payments settings"
-	cli wp option update woocommerce_woocommerce_payments_settings --format=json '{"enabled":"yes"}'
+	cli wp option set woocommerce_woocommerce_payments_settings --format=json '{"enabled":"yes"}'
```

Comment:
Just for my understanding, does `update` also fail similar to `add`?

Reply:
> Could we use shop_coupon get maybe to check and create if it doesn't exist? Maintaining raw queries could get trickier compared to standard CLI commands.

@deepakpathania `shop_coupon get` returns the DB output directly, which will be hard to parse the ID from to use with `shop_coupon delete`. I've already tried that but this way it'll ensure no coupons exists on the DB.

> Just for my understanding, does update also fail similar to add?
It doesn't fail, but I couldn't be sure about what happens if the meta row doesn't exist, so I also converted it to `set`, which the name suggests that it will both do `add` and `update` work.